src/Hosting/Hosting/src/WebHostExtensions.cs

```diff
@@ -114,8 +116,8 @@ private static async Task RunAsync(this IWebHost host, CancellationToken token,
 
                 if (!options.SuppressStatusMessages)
                 {
-                    Console.WriteLine($"Hosting environment: {hostingEnvironment.EnvironmentName}");
-                    Console.WriteLine($"Content root path: {hostingEnvironment.ContentRootPath}");
+                    Console.WriteLine($"Hosting environment: {hostingEnvironment?.EnvironmentName}");
```

Comment:
How can this be null?

Reply:
RunAsync does a `GetService`:

`var hostingEnvironment = host.Services.GetService<IHostEnvironment>();`

Reply:
Wow, bug 😄